server/app/assets/javascripts/admin_predicate_edit.ts

```diff
@@ -107,6 +112,31 @@ export class AdminPredicateEdit {
     )
   }
 
+  /**
+   * Depending on whether the user has added conditions in the predicate screen:
+   *    * If yes, then show the normal "Applicant is eligible / Screen is visible if any/all conditions are true" text
+   *    * If no, show the null state text.
+   */
+  private static showNodeOperatorSelectOrNullState(): void {
+    const nodeOperatorSelect = document.getElementById(
+      AdminPredicateEdit.NODE_OPERATOR_SELECT_ID,
+    )
+    if (nodeOperatorSelect) {
+      const nodeOperatorSelectNullState = assertNotNull(
+        document.getElementById(
+          AdminPredicateEdit.NODE_OPERATOR_SELECT_NULL_STATE_ID,
+        ),
+      )
+      if (document.querySelector('[data-testid="condition-1"]')) {
```

Comment:
(blocking) `data-testid` should only be used for Playwright in the browser tests when there are no alternative ways of selecting the element.